source/material_model/visco_plastic.cc

```diff
@@ -379,22 +411,58 @@ namespace aspect
                              Patterns::List(Patterns::Double(0)),
                              "List of thermal diffusivities, for background material and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
-                             "If only one values is given, then all use the same value.  Units: $m^2/s$");
+                             "If only one value is given, then all use the same value.  Units: $m^2/s$");
           prm.declare_entry ("Heat capacities", "1.25e3",
                              Patterns::List(Patterns::Double(0)),
                              "List of heat capacities $C_p$, for background material and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
-                             "If only one values is given, then all use the same value.  Units: $J/kg/K$");
+                             "If only one value is given, then all use the same value.  Units: $J/kg/K$");
           prm.declare_entry ("Densities", "3300.",
                              Patterns::List(Patterns::Double(0)),
                              "List of densities, $\\rho$, for background material and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
-                             "If only one values is given, then all use the same value.  Units: $kg / m^3$");
+                             "If only one value is given, then all use the same value.  Units: $kg / m^3$");
           prm.declare_entry ("Thermal expansivities", "3.5e-5",
                              Patterns::List(Patterns::Double(0)),
                              "List of thermal expansivities for background material and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
-                             "If only one values is given, then all use the same value.  Units: $1 / K$");
+                             "If only one value is given, then all use the same value.  Units: $1 / K$");
+
+          // Strain weakening parameters
+          prm.declare_entry ("Use strain weakening", "true",
```

Comment:
Do you prefer "true" to be the new default value? This will change user models that were done with this material model retroactively, usually we try to avoid such changes, unless they seem to be better in any case (but in this case it will change results unpredictably). I would prefer to have a "false" default. This would also remove the changes to the visco_plastic test output below. What do you think?